File: src/main/java/de/floribe2000/warships_java/direct/api/stats/OperationStatsContainer.java
package de.floribe2000.warships_java.direct.api.stats;

import java.util.HashMap;
import java.util.Map;
import lombok.Getter;

/**
 * Operation stats container extends the base with wins_by_tasks map.
 *
 * @author SirLefti
 */
@Getter
public class OperationStatsContainer extends BaseStatsContainer {

	private Map<Integer, Integer> wins_by_tasks = new HashMap<>();
}

File: src/main/java/de/floribe2000/warships_java/direct/api/stats/OperationStatsContainer.kt
package de.floribe2000.warships_java.direct.api.stats

import java.util.*

/**
 * Operation stats container extends the base with wins_by_tasks map.
 *
 * @author SirLefti
 */
class OperationStatsContainer : BaseStatsContainer() {
    val wins_by_tasks: Map<Int, Int> = HashMap()
}